src/localization/imu_packager.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+
+
+import rospy
+from geometry_msgs.msg import Vector3Stamped
+from mrover.msg import ImuAndMag
+from sensor_msgs.msg import Imu, MagneticField
+
+
+class ImuPackager:
+    """
+    This node is for use in sim - it merges the /imu/imu_only & /imu/mag_only messages
+    published by gazebo & publish to /imu/data, to replicate the behavior of imu_driver.py
+    """
+
+    imu_pub: rospy.Subscriber = None
```

Comment:
fyi, putting a member here makes it a class member not an instance member when the class isn't explicitly annotated as a dataclass. I would move these to init. Also if you initialize them to None the type isn't `rospy.Subscriber`, it's actually a `Optional[rospy.Subscriber]`

Reply:
yeah I would just put these in init and initialize them there since we don't really need them to be initialized to None before init runs